lib/wasi-types/src/wasi/wasix_manual.rs

```diff
@@ -0,0 +1,187 @@
+use std::mem::MaybeUninit;
+
+use wasmer::ValueType;
+
+use super::{
+    Errno, EventFdReadwrite, Eventtype, Signal, Snapshot0SubscriptionClock, SubscriptionClock,
+    SubscriptionFsReadwrite, Userdata,
+};
+
+pub type WasiHash = u128;
```

Comment:
FIXME: we are currently using C struct "de/serialization" for a few WASIX syscalls. 

This is mostly fine, but not for `u128`, which has different alignment requirements depending on the platform.
This should be replaced with 32 `u8` , or 2 `u64` or whatever.

Reply:
if you want to be free about any alignment, choose 32 `u8`, `u64` might have some on some platforms (especially 32bits)

Reply:
WasiHash was removed, because VBUS was removed completely.